<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSttCreateNewTestWidget</class>
 <widget class="QDialog" name="QSttCreateNewTestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建测试</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="1">
          <widget class="QLineEdit" name="m_edtDevIP">
           <property name="inputMask">
            <string>000.000.000.000; </string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="m_lblDeviceType">
           <property name="text">
            <string>设备类型</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="m_lblDevPort">
           <property name="text">
            <string>装置端口</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="m_lblProtolID">
           <property name="text">
            <string>规约引擎</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QScrollComboBox" name="m_cmbProtolID"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="m_cmbDeviceType"/>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="m_edtDevPort"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="m_lblDevIP">
           <property name="text">
            <string>装置IP</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="2">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayoutNameID" rowstretch="0,0,0,0">
       <item row="1" column="0">
        <widget class="QLabel" name="m_lblDvmFile">
         <property name="text">
          <string>模型文件</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="m_btnBrowsePp">
         <property name="text">
          <string>浏览</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="m_edtPpFile"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="m_btnBrowseDvm">
         <property name="text">
          <string>浏览</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="m_lblPpFile">
         <property name="text">
          <string>规约模板</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="m_edtDvmFile"/>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="m_btnSelFromScl">
         <property name="text">
          <string>从SCD文件选择</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="m_btnAdvancedCfg">
         <property name="text">
          <string>高级配置</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="m_chkTSettingUnit_ms">
         <property name="text">
          <string>时间定值单位为ms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_chkAddDevReset">
         <property name="text">
          <string>添加装置复归态</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_Btn_OK">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_Btn_Cancel">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QScrollComboBox</class>
   <extends>QComboBox</extends>
   <header>../Module/ScrollCtrl/ScrollComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
